lib/devil-bot-rust-cdk-stack.ts

```diff
@@ -1,9 +1,9 @@
-import { Construct, Stack, StackProps } from "@aws-cdk/core";
-import { Architecture, Code, Function, Runtime } from "@aws-cdk/aws-lambda";
-import { Policy, PolicyStatement } from "@aws-cdk/aws-iam";
-import { LambdaRestApi } from '@aws-cdk/aws-apigateway';
-import { Table, AttributeType } from '@aws-cdk/aws-dynamodb';
-import { RetentionDays } from "@aws-cdk/aws-logs";
+import {Construct, Stack, StackProps} from "@aws-cdk/core";
```

Comment:
What exactly is going on here? I like that import order is fixed, but I'm almost certain `prettier` auto formats to have leading/trailing spaces within the brackets (explicitly removed in this PR).